### ISSUE TITLE
add header to multiple files?

### ISSUE THREAD

**Issue by yairEO** [tiniest/headr#4]
I want to add it to a few files, is it possible in one short line of script?

Thanks

**Comment by ajiho** [comment]
Yes, it's possible see：https://github.com/wenzhixin/bootstrap-table/blob/develop/package.json#L61C25-L61C32
